### PR TITLE
fix: new import/export CLI

### PR DESCRIPTION
**Pull request by betodealmeida**
### SUMMARY

The new CLI for import/export is not working correctly with ZIP files (see https://github.com/apache/superset/issues/13914). When exporting a dataset or dashboard, the default output file name is `dashboard_export_YYYYMMDDTHHMMSS` (sic), while it should (1) have the `.zip` extension and (2) have an actual timestamp instead of the string "YYYYMMDDTHHMMSS".

Import is also broken for the new format, even with the `VERSIONED_EXPORT` feature flag on it assumes that the input file is the old YAML/JSON format used for datasets and dashboards, respectively. I fixed it by using the same logic as the API, inspecting the file to see if a ZIP file and unbundling it if that's the case.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TEST PLAN

1. Ran `superset export-dashboards`
2. Deleted all charts
3. Deleted all dashboards
4. Ran `superset import-dashboards -p dashboard_export_20210401T132948.zip`
5. Dashboards were imported correctly

Did the same with datasets (exported datasets; deleted all dashboards, charts, datasets; imported datasets).

Also added unit tests covering import/export with the feature flag on and off.

### ADDITIONAL INFORMATION

- [X] Has associated issue: https://github.com/apache/superset/issues/13914
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/cli.py [hunk 1]
```diff
@@ -266,6 +264,7 @@ def export_dashboards(dashboard_file: Optional[str]) -> None:
         timestamp = datetime.now().strftime("%Y%m%dT%H%M%S")
         root = f"dashboard_export_{timestamp}"
         dashboard_file = dashboard_file or f"{root}.zip"
+        print("\n\nBETO >>", dashboard_file)
```
Comment (eschutho) [inline superset/cli.py:267]: intentional print statement?

File: superset/cli.py [hunk 2]
```diff
@@ -250,13 +250,11 @@ def refresh_druid(datasource: str, merge: bool) -> None:
...
-        "--dashboard-file",
-        "-f",
-        default="dashboard_export_YYYYMMDDTHHMMSS",
-        help="Specify the the file to export to",
+        "--dashboard-file", "-f", help="Specify the the file to export to",
     )
-    def export_dashboards(dashboard_file: Optional[str]) -> None:
+    def export_dashboards(dashboard_file: Optional[str] = None) -> None:
         """Export dashboards to ZIP file"""
+        print("\nRIGHT WAS CALLED", dashboard_file)
```
Comment (eschutho) [inline superset/cli.py:257]: intentional?

Comment (betodealmeida) [reply superset/cli.py:257]: Oops

File: superset/cli.py (hunk 1 above)
Comment (betodealmeida) [reply superset/cli.py:267]: Oops

**Review by eschutho — APPROVED** [review]
Yey!

**Review by hughhhh — APPROVED** [review]

**Pull request merged**